lastpass-cli/README.md

```diff
@@ -0,0 +1,15 @@
+# lastpass-cli
+
+Allow command line interaction with LastPass service
+
+## Maintainers
+
+ - The Habitat Maintainers: humans@habitat.sh
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+Once installed the executable `lpass` will be available.
```

Comment:
Let's go a little further and explain the usage, how to install and execute.

```
hab pkg install core/lastpass-cli
hab pkg exec core/lastpass-cli lpass
```

etc.